[2025-05-12T08:07:57.668+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T08:07:57.701+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T08:07:55.735590+00:00 [queued]>
[2025-05-12T08:07:57.710+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T08:07:55.735590+00:00 [queued]>
[2025-05-12T08:07:57.711+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-12T08:07:57.730+0000] {taskinstance.py:2327} INFO - Executing <Task(BranchPythonOperator): check_if_retrain_needed> on 2025-05-12 08:07:55.735590+00:00
[2025-05-12T08:07:57.737+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_PWU', 'check_if_retrain_needed', 'manual__2025-05-12T08:07:55.735590+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/monitor_PWU.py', '--cfg-path', '/tmp/tmpot2fpe5t']
[2025-05-12T08:07:57.739+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask check_if_retrain_needed
[2025-05-12T08:07:57.741+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=399) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T08:07:57.742+0000] {standard_task_runner.py:63} INFO - Started process 400 to run task
[2025-05-12T08:07:57.781+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T08:07:55.735590+00:00 [running]> on host 6c14bac78271
[2025-05-12T08:07:57.865+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_PWU' AIRFLOW_CTX_TASK_ID='check_if_retrain_needed' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T08:07:55.735590+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T08:07:55.735590+00:00'
[2025-05-12T08:07:57.866+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T08:07:57.877+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-05-12T08:07:57.888+0000] {python.py:237} INFO - Done. Returned value was: trigger_retrain
[2025-05-12T08:07:57.888+0000] {branch.py:36} INFO - Branch into trigger_retrain
[2025-05-12T08:07:57.889+0000] {skipmixin.py:178} INFO - Following branch trigger_retrain
[2025-05-12T08:07:57.902+0000] {skipmixin.py:246} INFO - Skipping tasks [('end', -1)]
[2025-05-12T08:07:57.932+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T08:07:57.952+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_PWU, task_id=check_if_retrain_needed, execution_date=20250512T080755, start_date=20250512T080757, end_date=20250512T080757
[2025-05-12T08:07:57.996+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-12T08:07:58.022+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-12T08:07:58.027+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
